catalog/opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/impl/OpenSearchParserImpl.java

```diff
@@ -422,6 +500,10 @@ private boolean hasParameter(String parameter, List<String> parameters) {
     return new double[] {minX, minY, maxX, maxY};
   }
 
+  private static String createBboxParamString(double[] bboxCoords) {
+    return Arrays.stream(bboxCoords).mapToObj(Double::toString).collect(Collectors.joining(","));
```

Comment:
I also added some asserts in the unit tests to make sure that the bbox parameter contains the values is the correct west,south,east,north order.